frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/numberProperty.component.tsx

```diff
@@ -16,11 +16,11 @@
  */
 
 import { PropertyDefinition } from '@/v5/store/tickets/tickets.types';
-import { TextField } from '@mui/material';
+import { FormNumberField } from '@controls/formNumberField/formNumberField.component';
 
-export const NumberProperty = ({ property, value }: {property: PropertyDefinition, value:any}) => (
-	<div>
-	&nbsp;
-		<TextField label={property.name} value={value} disabled={property.readOnly} />
-	</div>
+export const NumberProperty = ({
+	property: { name, readOnly, required },
+	value,
```

Comment:
Same question here as before

Reply:
reference [this answer](https://github.com/3drepo/3drepo.io/pull/3696#discussion_r1001789280)